tktermwidget/tkterm.py

```diff
@@ -1,34 +1,37 @@
 """Terminal widget for tkinter"""
 from __future__ import annotations
 
-from os import getcwd
+from os import chdir, getcwd
 from pathlib import Path
 from platform import system
-from subprocess import PIPE, Popen
+from subprocess import CREATE_NEW_CONSOLE, PIPE, Popen
```

Comment:
This doesn't work on Max. It HAS to be in the if statement.

Reply:
```suggestion
from subprocess import PIPE, Popen
```